pkg/watch/health_test.go

```diff
@@ -117,7 +117,7 @@ output`))), nil)
 
```

Comment:
above, in the `[]byte`s, it seems that it would be more representative of a HEAD if there would be no body, but I suppose this tests that the body is never read...

I'm surprise the `val.Output` for the error cases below isn't tested